File: proxy/tests/script_engine/header_has.lua
Extensions = {
	{
		request = function(flow)
			if flow.request.headers:has("X-Header1") then
				flow.request.body.text = "has"
			end
		end,
		response = function(flow)
			if flow.response.headers:has("X-Header1") then
				flow.response.body.text = "has"
			end
		end,
	},
}

File: proxy/tests/script_engine/header_has.js
globalThis.extensions = [{
  request(flow) {
    if (flow.request.headers.has("X-Header1")) {
      flow.request.body.text = "has"
    }
  },
  response(flow) {
    if (flow.response.headers.has("X-Header1")) {
      flow.response.body.text = "has"
    }
  }
}];
